packages/@lwc/babel-plugin-component/src/__tests__/api-decorator.spec.js

```diff
@@ -55,23 +55,19 @@ describe('Transform property', () => {
         {
             output: {
                 code: `
-                import { registerDecorators as _registerDecorators2, registerDecorators as _registerDecorators, registerComponent as _registerComponent } from "lwc";
+                import { registerDecorators as _registerDecorators, registerComponent as _registerComponent } from "lwc";
                 import _tmpl from "./test.html";
 
                 class Outer {
                     outer;
-                    a = _registerDecorators2(
-                      class {
-                        innerA;
-                      },
-                      {
-                        publicProps: {
-                          innerA: {
-                            config: 0,
-                          },
+                    a = class {
+                      innerA;
+                      static publicProps = {
+                        innerA: {
+                          config: 0,
                         },
-                      }
-                    );
+                      };
+                    };
```

Comment:
@jodarove This test started failing after updating `@babel/core`. Looks like the inner class is no longer getting its decorators registered. Is this a breaking change?

Reply:
It's not a pattern we recommend, however, it's certainly a regression in the latest version of Babel with the update of `NodePath#_replaceWith`. I wasn't able to come up with a small repro to open a bug on Babel yet.

Were you able to isolate it @ekashida?